staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -242,12 +242,8 @@ func (s *GenericAPIServer) MinRequestTimeout() time.Duration {
 	return s.minRequestTimeout
 }
 
-type preparedGenericAPIServer struct {
```

Comment:
this deserves a comment: `// preparedGenericAPIServer is a private wrapper that enforces a call of PrepareRun() before Run can be invoked.`